modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -67,13 +67,24 @@ std::ostream& operator<< (std::ostream &os, const KernelPackage &e)
     return os;
 }
 
-struct GAPI_Streaming: public ::testing::TestWithParam<KernelPackage> {
-    GAPI_Streaming() { initTestDataPath(); }
+struct GAPI_Streaming: public ::testing::TestWithParam<std::tuple<KernelPackage,
+                                                                  cv::optional<size_t>>> {
```

Comment:
BTW `optional` doesn't print properly, It prints `0` if doesn't contain value.

I've tried to implement `operator<<` as for `KernelPackage` but it doesn't work :(

Reply:
Can you just use `compile_args`? `+=` should work for those.

Reply:
What this `compile_args` should contain ? I guess `queue_capacity` and `GKernelPackage`.

But for `SmokeTest_AutoMeta_VideoArray` need to exclude `GKernelPackage` and use `cv::gapi::kernels<TypesTest::OCVAddV>()` instead